scripts/release-branch.sh

```
@@ -149,6 +153,28 @@ pullRequest()
   echo "\n*** Pull request: $PR_URL"
 }
 
+confirmOnpremDirectMerge()
+{
+  echo ""
+      echo "*** Warning: merging $MAIN_BRANCH into $ONPREM_BRANCH may include in-progress HCCM and ROS"
+  echo "*** product work that has not been promoted to SaaS prod."
+  echo "*** Prefer scripts/release-onprem.sh, which pins those UIs to their latest prod tags."
+  echo ""
+  read -p "*** Continue with a direct $MAIN_BRANCH merge (y/N)? " YN
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)